code-challenges.test.js

```diff
@@ -1,51 +1,160 @@
+// // --------------------INSTRUCTOR EXAMPLE: Create a function that takes in an array of numbers and returns an array with all the numbers multiplied by 3.
 
-// a) Create a test with expect statements for each of the variables provided.
+// // a) Create a test with expect statements for each of the variables provided.
 
-const numbersArray1 = [6, 7, 8, 9, 10]
-// Expected output: [18, 21, 24, 27, 30]
-const numbersArray2 = [24, 27, 30, 33, 36]
-// Expected output: [72, 81, 90, 99, 108]
 
-// b) Create the function that makes the test pass.
+// describe ('multiplyByThree', () => {
+//     it('that takes in an array of numbers and returns an array with all the numbers multiplied by 3', () => {
+
+// const numbersArray1 = [6, 7, 8, 9, 10]
+// // Expected output: [18, 21, 24, 27, 30]
+// const numbersArray2 = [24, 27, 30, 33, 36]
+// // Expected output: [72, 81, 90, 99, 108]
+
+// expect(multiplyByThree(numbersArray1)).toEqual([18, 21, 24, 27, 30])
+// expect(multiplyByThree(numbersArray2)).toEqual([72, 81, 90, 99, 108])
+// })
+// })
+
+// // Output: ReferenceError: multiplyByThree is not defined
+
+// // b) Create the function that makes the test pass.
+
+// // Pseudo code:
+
+// //Input: an array of numbers
+// //Output: the product of the array of numbers multiplied by 3
+
+// const multiplyByThree = (arr) => {
+//     //use .map to iterate through array
+//     return arr.map((value) => {
+
+//         //multiply each iteration by 3, return product in new arrray
+//         return value * 3
+
+//     })
+// }
+
+//Output: PASS! that takes in an array of numbers and returns an array with all the numbers multiplied by 3
 
-// Pseudo code:
 
 // --------------------1) Create a function that takes a object as an argument and decides if the number inside it is evenly divisible by three or not.
 
 // a) Create a test with expect statements for each of the variables provided.
 
-const object1 = { number: 15 }
-// Expected output: "15 is divisible by three"
-const object2 = { number: 0 }
-// Expected output: "0 is divisible by three"
-const object3 = { number: -7 }
-// Expected output: "-7 is not divisible by three"
+// describe ('diviByThree', () => {
+//     it('takes a object as an argument and decides if the number inside it is evenly divisible by three or not', () => {
+    
+//         const object1 = {number: 15}
+//         expect(diviByThree(object1)).toEqual(15 + " " + 'is divisible by three')
+//         // Expected output: "15 is divisible by three"
+
+//         const object2 = {number: 0}
+//         expect(diviByThree(object2)).toEqual(0 + " " + 'is divisible by three')
+//         // Expected output: "0 is divisible by three"
+
+//         const object3 = {number: -7}
+        // expect(diviByThree(object3)).toEqual(-7 + " " + 'is not divisible by three')
+//         // Expected output: "-7 is not divisible by three"
```

Comment:
In your expect, you want exactly what is given in the expected output. You can just copy and paste the expected output into the parentheses after toEqual